### PR TITLE
Adding Azure instructions

### PR DESCRIPTION
**Pull request by srinarayan-srikanthan**
Added a tab for Azure and condensed two tabs to one.

### CODE REVIEW

**Comment by mkbhanda** [comment]
@srinarayan-srikanthan  is there a way to include some ignores in the hyperlink tests? In particular for links that have an access token that we naturally cannot include to pass the failed test?

**Comment by srinarayan-srikanthan** [comment]
@mkbhanda , yes looking into the fix for that.

**Comment by mkbhanda** [comment]
Further, that Azure home page is very generic .. taking the user through a lot of questions .. unless of course that is the only way.

**Review by mkbhanda — CHANGES_REQUESTED** [review]
Let us be consistent, either use bold or double quotes of single quotes for commands/buttons/selection options. We currently have a mix.

File: getting-started/README.md [hunk 1]
```diff
@@ -34,31 +34,41 @@ Before moving forward, it's important to familiarize yourself with two key eleme
...
 
-7. Once the instance is running, create and attach a `Floating IP` to the instance. For more information visit [this](https://cloud.ibm.com/docs/vpc?topic=vpc-fip-working&interface=ui) site
+7. Choose the Allow selected ports in Inbound port rule section and select HTTP. 
+
+8. Click Next: Disk button and select OS disk size as 128GiB.  
 
-8. `ssh` into the instance using the floating IP (`ssh -i <key> ubuntu@<floating-ip>`)
+9. Click on Review + Create to launch the VM. 
 
+10. Click Go to resource -> Connect -> Connect -> SSH using Azure CLI. Accept the terms and then select "Configure + connect"
```
[the remark this answers is not in the document]
Comment (srinarayan-srikanthan) [reply getting-started/README.md:69]: yes

File: getting-started/README.md [hunk 2]
```diff
@@ -34,34 +33,42 @@ Before moving forward, it's important to familiarize yourself with two key eleme
...
+
+11.  Select Add rule at the bottom, and create a rule with type as Custom TCP , port range as 80 and source as 0.0.0.0/0 . Learn more about [editing inbound/outbound rules](https://docs.aws.amazon.com/finspace/latest/userguide/step5-config-inbound-rule.html)
 
 :::
-:::{tab-item} IBM Cloud
-:sync: IBM Cloud
+:::{tab-item} Azure :sync: azure
 
-1.  Navigate to [IBM Cloud](https://cloud.ibm.com). - Click the **Create resource** button at the top right of the screen. Select **Compute** from the options available and select `Virtual Server for VPC`
+1. Navigate to [Microsoft Azure](portal.azure.com) – Select the "Skip" button on the bottom right to land on the service offerings page. Search for "Virtual Machines" in the search bar and select it. Click the "Create" button and select "Azure Virtual Machine".
```
Comment (chensuyue) [inline getting-started/README.md:47]: Please use `https://portal.azure.com` instead of `portal.azure.com`.

**Comment by chensuyue** [comment]
We will handle this issue in CI.

**Review by srinarayan-srikanthan — COMMENTED** [review]
done with all changes @mkbhanda

**Review by tomlenth — APPROVED** [review]

**Review by preethivenkatesh — APPROVED** [review]

File: getting-started/README.md [hunk 3]
```diff
@@ -14,73 +14,105 @@ Before moving forward, it's important to familiarize yourself with two key eleme
...
 
 2. Provide a name to the VM. 
 
-3. Select the base OS as Ubuntu 24.04 LTS from Marketplace . 
+3. Select the base OS as `Ubuntu 24.04 LTS` from Marketplace . 
 
 4. Select an Instance type that is based on Intel hardware. 
 
-> **Note:**  We recommend selecting a c3-standard-22 or larger instance with an Intel(R) 4th Gen Xeon(C) Scalable Processor, and the minimum supported c3 instance type is c3-standard-8 with 32GB memory. For more information on virtual servers on GCP visit [here](https://cloud.google.com/intel). 
+> **Note:**  We recommend selecting a `c3-standard-22` or larger instance with an Intel(R) 4th Gen Xeon(C) Scalable Processor, and the minimum supported c3 instance type is c3-standard-8 with 32GB memory. For more information on virtual servers on GCP visit [here](https://cloud.google.com/intel).
```
Comment (mkbhanda) [inline getting-started/README.md:81]:
```suggestion
> **Note:**  We recommend selecting a `c3-standard-22` or larger instance with an Intel(R) 4th Gen Xeon(C) Scalable Processor, and the minimum supported c3 instance type is c3-standard-8 with 32GB memory. For more information, visit [virtual servers on GCP](https://cloud.google.com/intel). 

```

File: getting-started/README.md [hunk 4]
```diff
@@ -144,53 +176,23 @@ Run `docker ps -a` as an additional check to verify that all the services are ru
...
 ![Chat Interface](assets/chat_ui_response.png)
 
+Given that any information about OPEA was not in the training data for the model, we see the model hallucinating and coming up with a response. We can upload a document (PDF) with information and observe how the response changes.
+
+> **Note:** this example leverages the [OPEA document](assets/what_is_opea.pdf) for its RAG based content. You can download the [OPEA document](assets/what_is_opea.pdf) and upload it using the UI. 
+
+![Chat Interface with RAG](assets/chat_ui_response_rag.png)
 
-> **Note:** this example leverages the Nike 2023 Annual report for its RAG based content. See the [ChatQnA Sample Guide](https://opea-project.github.io/latest/examples/ChatQnA/ChatQnA_Guide.html)
+We observe that the response is relavant and is based on the PDF uploaded.See the [ChatQnA Sample Guide](https://opea-project.github.io/latest/examples/ChatQnA/ChatQnA_Guide.html)
```
Comment (mkbhanda) [inline getting-started/README.md:195]:
```suggestion
We observe that the response is relevant and is based on the PDF uploaded. See the [ChatQnA Sample Guide](https://opea-project.github.io/latest/examples/ChatQnA/ChatQnA_Guide.html)

```

**Review by mkbhanda — CHANGES_REQUESTED** [review]
2 minor edits suggested.

**Review by mkbhanda — APPROVED** [review]
LGTM

**Pull request merged**